nixos/modules/profiles/minimal.nix

```diff
@@ -16,4 +16,8 @@ with lib;
   documentation.nixos.enable = mkDefault false;
 
   programs.command-not-found.enable = mkDefault false;
+
+  security.polkit.enable = mkDefault false;
+
+  services.udisks2.enable = mkDefault false;
```

Comment:
Does gparted and similar tools still work then?

Reply:
Gparted is used in the graphical ISO image. The `minimal.nix` profile is not used by them